ria_remote/create_sibling_ria.py

```diff
@@ -0,0 +1,230 @@
+    Furthermore, the former is configured to have a publication dependency on the latter.
+
+    Note, that the RIA remote needs to be configured before, referring to the name of the storage-sibling.
+    That is, access to it must be available via the 'annex.ria-remote.<STORAGE>.base-path' and optionally
+    'annex.ria-remote.<STORAGE>.ssh-host' configs.
+    """
+
+    _params_ = dict(
+        dataset=Parameter(
+            args=("-d", "--dataset"),
+            doc="""specify the dataset to process.  If
+            no dataset is given, an attempt is made to identify the dataset
+            based on the current working directory""",
+            constraints=EnsureDataset() | EnsureNone()),
+        sibling=Parameter(
+            args=("sibling",),
+            metavar="SIBLING",
+            doc="""name of the to be created sibling""",
+            constraints=EnsureStr() | EnsureNone()),
+        storage_sibling=Parameter(
+            args=("-s", "--storage-sibling"),
+            metavar="STORAGE",
+            doc="""name of the RIA storage sibling. Must not be identical to SIBLING. By default SIBLING is appended 
+            with '-storage'""",
+            constraints=EnsureStr() | EnsureNone()),
+        force=Parameter(
+            args=("-f", "--force"),
+            doc="""don't fail on existing siblings. Use and possibly reconfigure them instead.""",
+            action='store_true'),
+        no_publish=Parameter(
+            args=("--no-publish",),
+            doc="""whether to publish the dataset's history (no data) to SIBLING after creation.""",
+            action="store_true"),
+    )
+
+    @staticmethod
+    @datasetmethod(name='create_sibling_ria')
+    @eval_results
+    def __call__(
+            sibling,
+            dataset=None,
+            storage_sibling=None,
+            force=False,
+            no_publish=False):
+
+        # TODO: is check_installed actually required?
+        ds = require_dataset(dataset, check_installed=True, purpose='create sibling RIA')
+        res_kwargs = dict(
+            ds=ds,
+            action="create-sibling-ria",
+            logger=lgr,
+        )
+
+        if ds.repo.get_hexsha() is None or ds.id is None:
+            raise RuntimeError(
+                "Repository at {} is not a DataLad dataset, "
+                "run 'datalad create' first.".format(ds.path))
+
+        if not storage_sibling:
+            storage_sibling = "{}-storage".format(sibling)
+
+        if sibling == storage_sibling:
+            # leads to unresolvable, circular dependency with publish-depends
+            raise ValueError("sibling names must not be equal")
+
+        # TODO: messages - this is "create-sibling". Don't confuse existence of local remotes with existence of the
+        #       actual remote sibling in wording
+        if not force and sibling in [r['name'] for r in ds.siblings()]:
```

Comment:
Should disable the result rendering. Otherwise:

```
% datalad create-sibling-ria storedataladorg
.: here(+) [git]
.: here(+) [git]
...
```

Reply:
Done.